docs/skopeo.1.md

```diff
@@ -47,7 +47,12 @@ Most commands refer to container images, using a _transport_`:`_details_ format.
   **ostree:**_image_[**@**_/absolute/repo/path_]
   An image in local OSTree repository.  _/absolute/repo/path_ defaults to _/ostree/repo_.
 
-## OPTIONS
+## DOCKER REPOSITORY NAMES
+Docker Registry v2 API-compatible image registries support references of this type to the image namespaces themselves (e.g. docker.io/library/centos)
+
+  **registryhost:port/repositoryname** (e.g. docker.io/centos or docker.io/library/centos or quay.io/coreos/etcd)
```

Comment:
(The `:port` part might be missing, this would be nice to indicate here, and perhaps include an example like that.)

Reply:
Still outstanding.